src/app/frontend/zerostate/zerostate.controller.js

```diff
@@ -30,5 +30,14 @@ export default class ZeroStateController {
     $timeout(() => {
```

Comment:
You can now remove this "test" code, as there is actually something in there now :)